### ISSUE TITLE
Wrong variable used in ipabackup

### ISSUE THREAD

**Issue by FollowKenny** [freeipa/ansible-freeipa#513]
I'm trying to use the ipabackup role on a previously setup freeipa through the other roles.
I'm using ansible 2.10
When using ipabackup, the following error occurs :
`FAILED! => {
    "changed": false,
    "cmd": "/usr/bin/python3",
    "invocation": {
        "module_args": {
            "_raw_params": "/usr/bin/python3",
            "_uses_shell": false,
            "argv": null,
            "chdir": null,
            "creates": null,
            "executable": null,
            "removes": null,
            "stdin": "from ipaplatform.paths import paths\nprint(paths.IPA_BACKUP_DIR)\n",
            "stdin_add_newline": true,
            "strip_empty_ends": true,
            "warn": true
        }
    },
    "msg": "[Errno 2] No such file or directory",
    "rc": 2
}`

Indeed the remote host does not have python3 and in my opinion is not really supposed to. This is caused by the use of `{{ ansible_playbook_python }}` in [get_ipabackup_dir.yml](https://github.com/freeipa/ansible-freeipa/blob/master/roles/ipabackup/tasks/get_ipabackup_dir.yml). This variable is referencing the ansible controller's python and as I understand it has no reason to be used here.

I replaced the variable by `python` in the localy downloaded collection and everything worked fine. I don't know if this is the right fix but this variable should be replaced in any case.

**Steps to reproduce :**
- Setup a freeipa on Centos 7 using the default python 2.7 provided and the freeipa roles for the ansible controller (2.10 with python 3 used by default by the controller when launching ansible)
- try to backup the remote freeipa using the backup role provided

**Comment by t-woerner** [comment]
This should have been `ansible_python_interpreter`

**Comment by FollowKenny** [comment]
That was my first thought but if this variable is not specified in the inventory (or elsewhere) it is not set with a default value (it was not clear from the doc but I tried it before hardcoding `python` in the local collection and the variable was not found). I'm not sure if you are willing to introduce this dependancy and force the user to set the variable.

**Comment by t-woerner** [comment]
This should do it:

```
- command: "{{ ansible_playbook_python }}"
+ command: "{{ ansible_python_interpreter | default('/usr/bin/python') }}"
```

`ansible_python_interpreter` is only set if it is needed, we need to use the default if it is not set.

**Comment by FollowKenny** [comment]
Indeed did not think of that... Tested it it's working, if I make the PR, what's the release cycle (meaning when can it be merged in the main branch? ) ?

**Comment by FollowKenny** [comment]
Well I made the PR as it was really quick -> #514 Please keep me posted if this can be merged quickly or not !

**Comment by FollowKenny** [comment]
Hi, any update on the merge of the related PR ? I saw another PR fixing the bug in the tests but no news after

**Comment by rjeffman** [comment]
Closing as it was fixed by PR #514.
